pkgs/development/compilers/ante/default.nix

```diff
@@ -0,0 +1,60 @@
+{ fetchFromGitHub
+, lib
+, libffi
+, libxml2
+, llvmPackages_13
+, ncurses
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "ante";
+  version = "unstable-2022-07-24";
+  src = fetchFromGitHub {
+    owner = "jfecher";
+    repo = "ante";
+    rev = "187c284ecd3d8d60881f2f55feef47a051654c71";
+    sha256 = "sha256-drNEUISPgVZSFYjWh+yNMEx3aL+lKRQl8nQAlWRogbY=";
+  };
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

Comment:
Seems like upstream just committed a `Cargo.lock`.

Reply:
Yes, I asked the upstream maintainer for that, and they did it just after I made my PR.